cms/templatetags/cms_js_tags.py

```diff
@@ -45,10 +45,20 @@ def render_placeholder_toolbar_js(placeholder, render_language, content_renderer
     )
 
     def _render_plugin_js(plugin):
+        if plugin.plugin_type in plugin_children:
+            child_classes = plugin_children[plugin.plugin_type]
+        else:
+            child_classes = plugin.plugin_class.get_child_classes(slot=slot, page=page, instance=plugin)
```

Comment:
I believe this is an `AttributeError`.. plugin instances don't have a `plugin_class` attribute.